app/views/whats_new/_back_to_top.html.erb

```diff
@@ -0,0 +1,8 @@
+<p class="govuk-body-s app-view-whats-new__back-to-top-link">
```

Comment:
This is making use of classes `app-view-whats-new__back-to-top-link` that aren't defined in CSS